core/domain/feedback_services.py

```diff
@@ -315,66 +311,65 @@ def get_next_page_of_all_feedback_messages(
+        tuple(messages_on_page, next_urlsafe_start_cursor, more), where:
+            messages_on_page: list(FeedbackMessage). Contains the slice of
+                messages that are part of the page pointed to by the given start
+                cursor.
+            next_urlsafe_start_cursor: str. The cursor to the next page.
             more: bool. Whether there are more messages available to fetch after
                 this batch.
     """
-    results, new_urlsafe_start_cursor, more = (
+    models_on_page, next_urlsafe_start_cursor, more = (
         feedback_models.GeneralFeedbackMessageModel.get_all_messages(
             page_size, urlsafe_start_cursor))
-
-    result_messages = [_get_message_from_model(m) for m in results]
-    return (result_messages, new_urlsafe_start_cursor, more)
+    messages_on_page = [_get_message_from_model(m) for m in models_on_page]
+    return (messages_on_page, next_urlsafe_start_cursor, more)
 
 
 def get_thread_analytics_multi(exploration_ids):
     """Fetches all FeedbackAnalytics, for all the given exploration ids.
 
-    A FeedbackAnalytics contains the exploration id the analytics
-    belongs to, how many open threads exist for the exploration,
-    how many total threads exist for the exploration.
+    A FeedbackAnalytics contains the exploration id the analytics belongs to,
+    how many open threads exist for the exploration, how many total threads
+    exist for the exploration.
 
     Args:
-        exploration_ids: list of str. A list of exploration ids.
+        exploration_ids: list(str). A list of exploration ids.
 
     Returns:
-        list of FeedbackAnalytics. It's in the the same order as the input
-        list. If the exploration id is invalid, the number of threads in the
-        corresponding FeedbackAnalytics object will be zero.
+        list(FeedbackAnalytics). It's in the the same order as the input list.
+            If the exploration id is invalid, the number of threads in the
+            corresponding FeedbackAnalytics object will be zero.
     """
     return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics_multi( # pylint: disable=line-too-long
         exploration_ids)
 
 
 def get_thread_analytics(exploration_id):
-    """Fetches the FeedbackAnalytics for the given exploration id.
+    """Fetches the FeedbackAnalytics for the given exploration.
 
     Args:
-        exploration_id: str.
+        exploration_id: str. The id of the exploration.
 
     Returns:
-        list of FeedbackAnalytics.
+        FeedbackAnalytics. The feedback analytics of the given exploration.
     """
     return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics( # pylint: disable=line-too-long
         exploration_id)
 
 
-def get_total_open_threads(feedback_thread_analytics):
-    """Gets the count of all open threads for the given FeedbackThreadAnalytics
-    domain object.
+def get_total_open_threads(feedback_analytics):
```

Comment:
Should probably make this a method on the object?

Reply:
No, I mean a helper on the FeedbackAnalytics object. I.e. if the thing that's being passed in as an arg is a FeedbackAnalytics domain object, then we should be able to call feedback_analytics.get_total_open_threads() on it, right?

Reply:
Ah gotcha, so this is a method that acts on a _list_ of the domain objects, so the best we can do is make it a class method.